lab_1_keywords_tfidf/main.py

```diff
@@ -48,7 +62,18 @@ def calculate_frequencies(tokens: list[str]) -> Optional[dict[str, int]]:
 
     In case of corrupt input arguments, None is returned
     """
-    pass
+    if not (tokens and isinstance(tokens, list)):
+        return None
+
+    frequencies_dictionary = {}
+    for element in tokens:
+        if not isinstance(element, str):
+            return None
+        if element in frequencies_dictionary.keys():
```

Comment:
it is correct. yes
but given the fact that you aim at 8, try to find a way to compose a frequency dictionary using only one branch (or even less)  

in other words, how can we avoid using `else`?

Reply:
ok, done